src/ResourceManager/Network/Commands.Network/VirtualNetworkGatewayConnection/UpdateAzureVirtualNetworkGatewayConnectionCommand.cs

```diff
@@ -36,13 +36,24 @@ public class SetAzureVirtualNetworkGatewayConnectionCommand : VirtualNetworkGate
         [Parameter(
             Mandatory = false,
             ValueFromPipelineByPropertyName = true,
-            HelpMessage = "Whether to establish a BGP session over a S2S VPN tunnel")]
-        public string EnableBgp { get; set; }
+            HelpMessage = "Enable a BGP session over a S2S VPN tunnel")]
+        public SwitchParameter EnableBgp { get; set; }
 
         [Parameter(
             Mandatory = false,
-            HelpMessage = "Use policy-based traffic selectors for a S2S connection")]
-        public string UsePolicyBasedTrafficSelectors { get; set; }
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Disable a BGP session over a S2S VPN tunnel")]
+        public SwitchParameter DisableBgp { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Enable policy-based traffic selectors for a S2S connection")]
+        public SwitchParameter EnablePolicyBasedTrafficSelectors { get; set; }
```

Comment:
Since there are multiple switches here, please change this to bool parameters, or set up parameter sets to make these switch parameters mutually exclusive, per the advies in the email about this.  Either would be acceptable